snippets/quick-order-list-row.liquid

```diff
@@ -159,12 +159,10 @@
       </div>
     {%- endif -%}
   </td>
-  <td class="variant-item__totals right large-up-hide">
+  {% assign total_item_price_in_cart = cart | line_items_for: item | sum: 'original_line_price' %}
+  <td class="variant-item__totals{% if total_item_price_in_cart == 0 %} greyed-out{% endif %} right large-up-hide">
```

Comment:
`greyed-out` feels like a name of a generic class that is used elsewhere, but this is only for quick order form. Can we update the name here? What about this?

```suggestion
  <td class="variant-item__totals{% if total_item_price_in_cart == 0 %} variant-item__totals--disabled{% endif %} right large-up-hide">
```